frontend/src/components/common/EditorDialog.tsx

```diff
@@ -293,15 +293,26 @@ function DocsViewer(props: {docSpecs: any}) {
   const { docSpecs } = props;
   const classes = useStyles();
   const [docs, setDocs] = React.useState<object | null>(null);
-
```

Comment:
Please don't remove newlines like this. You should add your new state line to the top of the newline.